src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -508,6 +508,7 @@ public void testAllCheckSections() throws Exception {
      * Test contains asserts in callstack, but idea does not see them.
```

Comment:
Based on similar other change, shouldn't this text be removed?

Reply:
Done.